osu.iOS.props

```diff
@@ -75,4 +75,17 @@
     <PackageReference Include="ppy.osu.Game.Resources" Version="2019.1010.0" />
     <PackageReference Include="ppy.osu.Framework.iOS" Version="2019.1112.0" />
   </ItemGroup>
+  <!-- Xamarin.iOS does not automatically handle transitive dependencies from NuGet packages. -->
+  <ItemGroup Label="Transitive Dependencies">
```

Comment:
how has it been working until now?

Reply:
@huoyaoyuan removed these explicit dependencies from osu.iOS.props assuming they were unrequired.  My commit readded them.

Reply:
they aren't in master – does that mean master is broken?

Reply:
Yes. It's not effectively verified in CI.